src/main/java/at/hannibal2/skyhanni/utils/StringUtils.kt

```diff
@@ -146,6 +146,10 @@ object StringUtils {
         }
     }
 
+    fun Int.trailingS(): String {
```

Comment:
Terrible name. Also I would just use `canBePlural`/`optionalPlural` (what the hell are these names?), maybe extend it to to default to `${singular}s`.

Reply:
I prefer the original name: `s`

Reply:
#975

Reply:
https://git.nea.moe/nea/neuhax/src/commit/c2467b478705ccb3a2d0800beff9b1b4895c43c3/src/main/kotlin/moe/nea/sky/util/chatutils.kt#L83

Reply:
seeing as we're not playing code golf, #975